src/app/_components/StackingSection/StackingSection.tsx

```diff
@@ -117,7 +117,9 @@ function CycleHeader() {
           fontWeight={['medium', 'regular']}
           whiteSpace={'nowrap'}
         >
-          Ends in ~{approximateDaysTilNextCycle} day{approximateDaysTilNextCycle > 1 ? 's' : ''}
+          {approximateDaysTilNextCycle < 1
+            ? 'Ends today'
+            : `Ends in ~${approximateDaysTilNextCycle} days`}
```

Comment:
If approximateDaysTilNextCycle === 1 we'd want to display `1 day` instead of `1 days`